### PR TITLE
London-8 /Alireza Bagheri

### PR DESCRIPTION
**Pull request by Alirezabg**
**Volunteers: Are you marking this coursework?** _You can find a guide on how to mark this coursework in `HOW_TO_MARK.md` in the root of this repository_

# Your Details

- Your Name:
- Your City:
- Your Slack Name:

# Homework Details

- Module:
- Week:

# Notes

- What did you find easy?

- What did you find hard?

- What do you still not understand?

- Any other notes?

-----
[View rendered .github/pull_request_template.md](https://github.com/Alirezabg/JavaScript-Core-1-Coursework-Week1/blob/master/.github/pull_request_template.md)

### CODE REVIEW

File: extra/1-currency-conversion.js [hunk 1]
```diff
@@ -15,8 +16,10 @@ function convertToUSD() {}
   They have also decided that they should add a 1% fee to all foreign transactions, which means you only convert 99% of the £ to BRL.
 */
 
-function convertToBRL() {}
-
+function convertToBRL(brazilianGbp) {
```
Comment (Selchuk-Karakus) [inline extra/1-currency-conversion.js:19]: Your parameter naming makes this functions purpose difficult to understand. Try to use meaningful/semantic naming throughout your code so when others look at your code they have an idea of what it's supposed to do.  I would suggest using  "pound" or "amount".

File: java.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+function addNumbers(a, b, c) {
+  return a + b + c;
+}
+console.log(addNumbers(3, 4, 6));
+
+function introduceMe(name, age) {
+  return `Hello, my name is ${name} and I am ${age} years old`;
```
Comment (Selchuk-Karakus) [inline java.js]: Great use of string interpolation!

File: mandatory/4-tax.js [hunk 3]
```diff
@@ -5,7 +5,9 @@
   Sales tax is 20% of the price of the product.
 */
 
-function calculateSalesTax() {}
+function calculateSalesTax(saleTax) {
+  return saleTax*1.2;
```
Comment (Selchuk-Karakus) [inline mandatory/4-tax.js:9]: It looks like you've left out the space on either side of the multiplication math operator.  It makes this line of code unclear and hard to read.

**Review by Selchuk-Karakus — CHANGES_REQUESTED** [review]
Great work overall!

File: extra/1-currency-conversion.js (hunk 1 above)
Comment (Alirezabg) [reply extra/1-currency-conversion.js:19]: Thanks

**Pull request closed without merging**